lib/sms.rb

```diff
@@ -0,0 +1,24 @@
+require 'twilio-ruby'
+
+class Sms
+
+  def initialize
+    @client = twilio_client
+  end
+
+  def send_sms(message)
+    @client.messages.create(
+     from: '+441424531106',
+     to: '+447789515988',
+     body: message
+   )
```

Comment:
Style/ClosingParenthesisIndentation: Indent `)` the same as the start of the line where `(` is.